src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -122,6 +125,12 @@ displayUserError e = case e of
       "The bower.json file was not found. Please create one, or run " ++
       "`pulp init`."
       )
+  LicenseNotFound ->
+    para (concat
+      ["A LICENSE file was not found. Please create one. ",
+       "Distributing code without a license means that nobody",
+       "will be able to (legally) use it."
```

Comment:
Also maybe "No LICENSE file was found" rather than "A LICENSE file was not found"?